test/category.test.ts

```diff
@@ -4,12 +4,13 @@ import Database from '../src/database';
 import chaiHttp from 'chai-http';
 import { describe, it } from 'mocha';
 import app from '../src/app';
-import { token } from './product.test';
+import { headerTokenSeller } from './2FA.tets';
+
 
```

Comment:
I know now why test are failing this you changed something that you don't even know how it's works

Reply:
done

Reply:
Fabrice test are not failing  the token i imported form my test is the valid token you get when you are seller please check 2FA.tets.ts